draft-law-moq-warpstreamingformat.md

```diff
@@ -73,7 +64,7 @@ This document specifies the WARP Streaming Format, designed to operate on Media
 
 # Introduction
 
-WARP Streaming Format (WARP) is a media format designed to deliver CMAF {{CMAF}} and LOC {{LOC}} compliant media content over Media Over QUIC Transport (MOQT) {{MoQTransport}}. WARP works by fragmenting the bitstream into objects that can be independently transmitted. WARP leverages the Common Catalog Format {{COMMON-CATALOG-FORMAT}} to describe the output of the original publisher. WARP specifies how content should be packaged and signaled, defines how the catalog communicates the content, specifies prioritization strategies for real-time and workflows for beginning and terminating broadcasts. WARP also details how end-subscribers may perform adaptive bitrate switching. WARP is targeted at real-time and interactive levels of live latency.
+WARP Streaming Format (WARP) is a media format designed to deliver LOC {{LOC}} compliant media content over Media Over QUIC Transport (MOQT) {{MoQTransport}}. WARP works by fragmenting the bitstream into objects that can be independently transmitted. WARP leverages the Common Catalog Format {{COMMON-CATALOG-FORMAT}} to describe the output of the original publisher. WARP specifies how content should be packaged and signaled, defines how the catalog communicates the content, specifies prioritization strategies for real-time and workflows for beginning and terminating broadcasts. WARP also details how end-subscribers may perform adaptive bitrate switching. WARP is targeted at real-time and interactive levels of live latency.
```

Comment:
I already update this section with my PR to remove the CMCAF and Common Cataklog Format references, so we could omit this part as it is redundant if we re-base.